dokka-subprojects/plugin-versioning/src/main/kotlin/org/jetbrains/dokka/versioning/VersioningConfiguration.kt

```diff
@@ -13,7 +13,8 @@ public data class VersioningConfiguration(
     var olderVersions: List<File>? = defaultOlderVersions,
     var versionsOrdering: List<String>? = defaultVersionsOrdering,
     var version: String? = defaultVersion,
-    var renderVersionsNavigationOnAllPages: Boolean? = defaultRenderVersionsNavigationOnAllPages
+    var renderVersionsNavigationOnAllPages: Boolean? = defaultRenderVersionsNavigationOnAllPages,
+    var olderVersionsDirName: String = defaultOlderVersionsDirName
```

Comment:
Should `olderVersionsDirName` be nullable like other optional parameters?

Reply:
AFAIU it shouldn’t be `nullable` - it's already `optional` because it has a default (`"older"`). The same pattern is used in [DokkaBaseConfiguration.footerMessage](https://github.com/Kotlin/dokka/blob/3a0a602c2fd87666116c5c6b7a1a6616a7e62a35/dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/DokkaBaseConfiguration.kt#L15)

With `null` value, IMO, it is a bit confusing on what should happen in absence of the value (especially when it is coming from `json` configuration): if it is absent - should we use `default` value or not create a folder or fail or anything else.